app/cypress.config.ts

```diff
@@ -1,4 +1,10 @@
 import { defineConfig } from 'cypress';
+import * as dotenv from 'dotenv';
+
+function getBaseUrlFromEnv() {
+    dotenv.config();
+    return process.env.CYPRESS_BASE_URL;
+}
```

Comment:
pretty sure this shouldn't be necessary - cypress will already automatically populate the baseUrl when CYPRESS_BASE_URL is set. See [here](https://docs.cypress.io/guides/guides/environment-variables#We-can-move-this-into-a-Cypress-environment-variable)

Reply:
You are right. This part is mainly for reading the CYPRESS_BASE_URL from a local `.env` file. (dotenv.config() does that), so that we don't need to manually set the CYPRESS_BASE_URL when running in local.